test/e2e/app/main.go

```diff
@@ -62,11 +59,12 @@ func run(configFile string) error {
 	// Start app server.
 	switch cfg.Protocol {
 	case "builtin":
-		if len(cfg.Misbehaviors) == 0 {
-			err = startNode(cfg)
-		} else {
-			err = startMaverick(cfg)
-		}
+		// FIXME: Temporarily remove maverick until it is redesigned
```

Comment:
Ok, I see

> Maverick: I'll be back

Reply:
Does the tm team plan to redesign maverick or we? IMO: good redesign would require consensus redesign in a more modular way. Copying code is nah

Reply:
Yeah, it redesigned it already as far as I undertand. It's now part of the other e2e tests (at least the most relevant parts).